src/main/java/core/basesyntax/service/FruitService.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+
+public interface FruitService {
+    FruitDao FRUIT_DAO = new FruitDaoImpl();
+    void moveFruit(String fruit, Integer amount);
+}
```

Comment:
In general you have only 2 operations, so probably we should keep only 2 implementations?